.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,14 +1,62 @@
 ---
 name: Bug Report
-about: Report 🐞 Bugs
-title: ''
+about: Create a 🐞 Bug Report to Improve Athens
+title: 'Descriptive Title with Category'
 labels: 'type: 🐞 bug'
 assignees: ''
+ID number: ''
+Category: ''
 
 ---
 
-**Problem**
+**Before Beginning:**
+
+- [ ] Read [Contribution Guidelines](https://app.gitbook.com/@athensresearch/s/handbook/~/drafts/-MZt8Vmd14cTrIp7xfV9/contributing-1/code-of-conduct/@archived) and [Code of Conduct](https://app.gitbook.com/@athensresearch/s/handbook/~/drafts/-MZt8Vmd14cTrIp7xfV9/contributing-1/code-of-conduct/@archived)
```

Comment:
https://athensresearch.gitbook.io/handbook/contributing

Reply:
https://athensresearch.gitbook.io/handbook/contributing-1/code-of-conduct

I mistakenly pasted the urls from the version I was editing in Gitbook.

Reply:
Please fix the URLs in the code then. I still see the inaccessible URLs here.